### ISSUE TITLE
Building characteristics for upgrade rows

### ISSUE THREAD

**Issue by shorowit** [NREL/OpenStudio-BuildStock#86]
For upgrade rows in the results.csv, the building characteristics can be confusing. For example, if you replace a Furnace/AC with a MSHP, you'll see:

Datapoint|Furnace|AC|MSHP|
Baseline|80% AFUE|SEER 13|None|
Upgrade|80% AFUE|SEER 13|HSPF 10, SEER 20|

It's not really possible to fix when you consider all of the upstream parameters (fuel type, HVAC is combined, etc etc.). 

And for ComStock, it's not going to show a change in building characteristics at all since they use separate measures for retrofits.

In short, I think we should blank out the building characteristic rows in the results.csv. Not sure if this impacts @joseph-robertson's scripts or @nmerket's website?

cc @ejhw

**Comment by ejhw** [comment]
@shorowit Blank them out in post-processing, or not have OS report them in the first place?

**Comment by shorowit** [comment]
The latter. Blank them out in our building characteristics reporting measure.

**Comment by ejhw** [comment]
For all datapoints, or just the upgrades?

**Comment by shorowit** [comment]
Just the upgrades.

**Comment by ejhw** [comment]
That's good. It would mean there's not an easy way to see the post-retrofit characteristics as the website can currently display. Maybe that's fine.

How would this affect Reference scenarios though? Those are accomplished with upgrades, so there wouldn't be an easy way to see what characteristics are in the Reference scenario(s), right?

**Comment by shorowit** [comment]
I don't think we finalized our recommendation for how to implement Reference scenarios. It's true that one of our recommendations we discussed would be impacted by this -- so probably best to look for other approaches.
